oauth2-server/src/test/java/com/clouway/oauth2/token/TokenRequestEqualityTest.java

```diff
@@ -12,17 +20,164 @@
 public class TokenRequestEqualityTest {
   @Test
   public void areEqual() {
-    TokenRequest request1 = new TokenRequest("grant_type", "code","refresh_token", "client_id", "client_secret");
-    TokenRequest request2 = new TokenRequest("grant_type", "code","refresh_token", "client_id", "client_secret");
+    DateTime anyDateTime = new DateTime();
```

Comment:
This test looks a bit overwhelming with few instantiations of whole object. You can simplify it by using 2 cases: equal and not equal. 

EqualsVerifier looks like a library which could be tested about this: http://jqno.nl/equalsverifier/manual/getting-started/

Reply:
With the implementation of this library should i change the Equals tests for other classes as well or should i do it in a separate PR?

Reply:
In separate PR sounds better.